src/autostep/Program.cs

```diff
@@ -46,5 +47,13 @@ static async Task<int> Main(string[] args)
 
             return await result.InvokeAsync();
         }
+
+        private static bool DisplayHelp(string[] args) =>
```

Comment:
The naming of this method implies action (``Display``), rather than a check. Perhaps rename to ``ShouldDisplayHelp``?